src/features/authSlice.js

```diff
@@ -39,34 +39,26 @@ const postSignupDetails = createAsyncThunk(
   }
 );
 
-const initialState = {
-  username: "",
-  name: "",
-  bio: "",
-  profileImage: "",
-  profileCover: "",
-  bookmarks: [],
-  followers: [],
-  following: [],
-};
 const authSlice = createSlice({
   name: "authentication",
-  initialState,
+  initialState: {},
   reducers: {},
   extraReducers: {
     [postLoginDetails.pending]: () => {
       console.log("login pending");
     },
     [postLoginDetails.fulfilled]: (state, { payload }) => {
-      state.username = payload.data.user.username;
-      state.name = payload.data.user.name;
-      state.bio = payload.data.user.bio;
-      state.profileImage = payload.data.user.profileImage;
-      state.profileCover = payload.data.user.profileCover;
-      // state.bookmark = payload.user.bookmarks;
-      state.followers = payload.data.user.followers;
-      state.following = payload.data.user.following;
-      localStorage?.setItem("userToken", payload.data.token);
+      console.log(payload.data);
+      state = payload.data.data.user;
+      // state.username = payload.data.user.username;
+      // state.name = payload.data.user.name;
+      // state.bio = payload.data.user.bio;
+      // state.profileImage = payload.data.user.profileImage;
+      // state.profileCover = payload.data.user.profileCover;
+      // // state.bookmark = payload.user.bookmarks;
+      // state.followers = payload.data.user.followers;
+      // state.following = payload.data.user.following;
```

Comment:
can remove the commented code

Reply:
yes , thank you for suggesting  . I will do the changes